src/nvim/msgpack_rpc/helpers.c

```diff
@@ -385,43 +384,43 @@ void msgpack_rpc_from_object(const Object result, msgpack_packer *const res)
                   && kObjectTypeTabpage == kObjectTypeWindow + 1,
                   "Buffer, window and tabpage enum items are in order");
     switch (cur.aobj->type) {
-      case kObjectTypeNil:
-      case kObjectTypeLuaRef: {
+    case kObjectTypeNil:
+    case kObjectTypeLuaRef: {
```

Comment:
`{` in cases could be deleted if there's no scoped vars, to avoid the weirdly indented final `}`

Reply:
it's a chore to add the `{` later though. would be less churn to have 1 weirdly-indented line.

Reply:
I removed the extra indentation inside braces in case, I *think* that should solve the issue with the bad indentation of `}`. The fix affected a lot of other code though, so please check it didn't mess up anything else in the process.

If this doesn't work it's also possible to remove the braces if there are no variable declarations inside.